### PR TITLE
Check repeated tuple

### PR DESCRIPTION
**Pull request by kerolossamir165**
# Challenge [CheckRepeatedChars 27958]

## Checklist

- [ ] Make sure that you are opening the PR with a single solution (one solution per PR)
- [ ] The solution must have at least several references for the readers to dive in depth

### CODE REVIEW

**Review by ghaiklor — CHANGES_REQUESTED** [review]
It's not clear the path you took. I suggest splitting in a few more intermediate types, so it looks more incremental.

**Review by ghaiklor — COMMENTED** [review]
Please, use the grammar checker for English, you have a lot of mistakes, and it's hard to follow what you are trying to communicate. I can't merge it until it will be a well explained post about the solution. Thanks!

**Comment by kerolossamir165** [comment]
I tried my best to enhance the grammar and the explanation of the solution 😅.
